Dockerfile

```
@@ -1,13 +1,29 @@
+# Get qemu-user-static
+ARG IMAGEARCH
+FROM alpine:3.9.2 as qemu
+RUN apk add --no-cache curl
+ARG QEMUVERSION=2.9.1
```

QEMU version needs to be update
```suggestion
ARG QEMUVERSION=4.1.1-1
```